geth/signal/signals.go

```diff
@@ -47,14 +48,21 @@ type NodeNotificationHandler func(jsonEvent string)
 
 var notificationHandler NodeNotificationHandler = TriggerDefaultNodeNotificationHandler
 
+// notificationHandlerMutex needs to remove data races on SetDefaultNodeNotificationHandler,ResetDefaultNodeNotificationHandler
```

Comment:
I think a simpler comment like ".. guards `notificationHandler` for concurrent calls...` would be enough — no need to update methods names mentioned in a current comment, for example.